src/main/cpp/xmlhw/ColorSensorDefn.h

```diff
@@ -0,0 +1,21 @@
+
+
+#include <pugixml/pugixml.hpp>
+
+#include <ctre/Phoenix.h>
+#include <rev/ColorSensorV3.h>
+using namespace std;
```

Comment:
delete this; it isn't used here.  It is not good practice to have using namespace in the .h file.